internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -270,22 +276,24 @@ func (r *BMACReconciler) reconcileAgentInventory(bmh *bmh_v1alpha1.BareMetalHost
 
 	// Add storage
 	for _, d := range inventory.Disks {
-		// missing vendor, rotational, WWNVendorExtension, WWNWithExtension
+		// missing WWNVendorExtension, WWNWithExtension
 		disk := bmh_v1alpha1.Storage{
 			Name:         d.Path,
 			HCTL:         d.Hctl,
 			Model:        d.Model,
 			SizeBytes:    bmh_v1alpha1.Capacity(d.SizeBytes),
 			SerialNumber: d.Serial,
 			WWN:          d.Wwn,
+			Vendor:       d.Vendor,
+			Rotational:   strings.EqualFold(d.DriveType, "hdd"),
 		}
 
 		hardwareDetails.Storage = append(hardwareDetails.Storage, disk)
 	}
 
 	// Add the memory information in MebiByte
 	if agent.Status.Inventory.Memory.PhysicalBytes > 0 {
-		hardwareDetails.RAMMebibytes = int(inventory.Memory.PhysicalBytes / (1024 * 1024))
+		hardwareDetails.RAMMebibytes = int(conversions.BytesToGiB(inventory.Memory.PhysicalBytes))
```

Comment:
MiB on the left side, GiB on the right, is it intended? I think the earlier version used MiB as well.

Reply:
https://github.com/openshift/assisted-service/pull/4453